modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/java/com/liferay/frontend/editor/ckeditor/web/internal/editor/configuration/BaseCKEditorConfigContributor.java

```diff
@@ -40,12 +44,28 @@ public void populateConfigJSONObject(
 
 		jsonObject.put("allowedContent", Boolean.TRUE);
 
+		StringBundler sb = new StringBundler(5);
+
+		sb.append("cke_editable html-editor");
```

Comment:
Hi @markocikos, thank you for this solution! I was trying to do this in Forms side but it would bring a little bug for us hahaha I'm glad this fix can be done on CKEditor!! \o/